docs/monitoring.md

```diff
@@ -518,6 +518,15 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td><code>/applications/[app-id]/allexecutors</code></td>
     <td>A list of all(active and dead) executors for the given application.</td>
   </tr>
+  <tr>
+    <td><code>/applications/[app-id]/executorSummary</code></td>
+    <td>
+      Summary peak memory metrics of executors.
```

Comment:
this is misleading the peak memory metrics are in the executors rest api already right?
This seems just to add in the ability to get them as quantiles?